aptos-move/framework/aptos-framework/sources/AptosLib.move

```diff
@@ -0,0 +1,29 @@
+module AptosFramework::AptosLib {
+    struct TypeInfo has copy, drop {
```

Comment:
Can have `store` as well? Deliberately avoiding?

Reply:
I guess I don't add things without motivation for why. The info should already be available in context ... I think?